src/libraries/System.Text.RegularExpressions/gen/Stubs.cs

```diff
@@ -65,6 +65,14 @@ namespace System.Buffers
     internal delegate void SpanAction<T, in TArg>(Span<T> span, TArg arg);
 }
 
+namespace System.Numerics
+{
+    internal static class BitOperations
+    {
+        public static bool IsPow2(int value) => (value & (value - 1)) == 0 && value > 0;
```

Comment:
Is this needed because the source generator needs to work downlevel?

Reply:
I believe SG are supposed to target NS2.0. (But what happens if they don't? Is it possible they could get loaded in a compiler on older runtime than latest?)

Reply:
> But what happens if they don't? Is it possible they could get loaded in a compiler on older runtime than latest?

Yes, that's the common case. What's the concern?